enemies/bosses/main.json

```diff
@@ -22,6 +22,7 @@
         ]
       },
       "hp": 18000,
+      "expectedDamageFrames": 300,
```

Comment:
18000 hp at 20 hp per hit, 4 shots per second, adds up to a 225-second fight, or 13500 frames. Assuming contact damage, you now logically need 2 etanks to beat Ceres Ridley.

Reply:
I thought I changed it back to 0, doh